src/components/common/Header/Header.tsx

```diff
@@ -13,7 +13,10 @@ const Header = ({ menuItems = [] }: HeaderProps): React.ReactElement => {
     <>
       <header>
         <h2>Story Squad</h2>
-        <MdMenu onClick={toggleMenu} />
+        <MdMenu
+          className={menuItems.length <= 0 ? 'hidden' : ''}
```

Comment:
This hides the hamburger menu icon on mobile when there are no menu items passed into the Header